client/components/jetpack/licensing-prompt-dialog/index.tsx

```diff
@@ -45,9 +46,13 @@ function LicensingPromptDialog( { siteId }: Props ) {
 		dispatch( recordTracksEvent( 'calypso_user_license_modal_view' ) );
 	}, [ dispatch ] );
 
-	const title = useMemo( () => {
+	let titleToRender = isEnabled( 'jetpack/pricing-page-rework-v1' )
+		? translate( 'Jetpack is successfully installed' )
+		: '';
+
+	if ( ! titleToRender ) {
```

Comment:
Won't it always be true? `!'' === true`

Reply:
It will be so when the feature is not enabled.

Reply:
So when feature disabled it will be empty string `''` p- no?

Reply:
Correct and it will retain the existing behavior.

Reply:
Right, sorry, it's I was thinking of something else:)